tstest/main.ts

```diff
@@ -74,5 +74,8 @@ sub.close();
 sub.unsubscribe();
 
 
+console.table(NewSubscriptionOptions());
```

Comment:
No idea what that does: print to console? :-)

Reply:
all we are doing is verifying typescript is seeing the type.